### ISSUE TITLE
What's the purpose of the "View" button in notification popup?

### ISSUE THREAD

**Issue by ldmpub** [martyr-deepin/deepin-screenshot#42]
I'm running Xubuntu with deepin screenshot. When a screenshot has been taken a notification popup is raised with a "View" button.

What should occurs when user clicks on this button:
1. button closes the popup => it's the current behavior for me, if confirmed we need to change the button label to a more relevant "Close".
2. button opens the image captured => the button label is correct but it doesn't work / does not open image in my Xubuntu / XFCE environment ;-(.

Thanks for your answers,

**Comment by mikield** [comment]
@ldmpub can you provide a screenshot what do you mean buy button "View". Cause I can not find any kind of that button.

**Comment by ldmpub** [comment]
@mikield, here is a screenshot with the button I'm wondering what does it should do (currently it closes the notification popup):
![deepin capture-ecran_zone de selection _20181115221407](https://user-images.githubusercontent.com/1452146/48582085-de72e900-e923-11e8-9021-00ffe6a07149.png)

Please note: I have requested to join the french translation team on transifex, my request was accepted but I have not rights at all! not even to make suggestions ... and "moderators", contacted by transifex inmail, do not answer.

**Comment by ldmpub** [comment]
@mikield, here is a screenshot with the button I'm wondering what does it should do (currently it closes the notification popup):
![deepin capture-ecran_zone de selection _20181115221407](https://user-images.githubusercontent.com/1452146/48582085-de72e900-e923-11e8-9021-00ffe6a07149.png)

Please note: I have requested to join the french translation team on transifex, my request was accepted but I have not rights at all! not even to make suggestions ... and "moderators", contacted by transifex inmail, do not answer.

**Comment by ldmpub** [comment]
Hi, what is the wanted behaviour of above button in notification pop-up?

**Comment by ujdhesa** [comment]
@ldmpub, it's string number three in https://www.transifex.com/linuxdeepin/deepin-screenshot/viewstrings/#fr/deepin-screenshot-ts/113745852?q=translation_text%3AVue . The original says __View__ and from the context of your screenshot it looks like through that button you can view the screenshot you just did. Obviously it's describing an action, so (from my rudimentary French) I find Vue kind of wrong.

**Comment by ldmpub** [comment]
Thanks. I agree: if it's an action it should be a verb instead of a noun in all languages.

The real question is: **What does a click on this button do?**
Can someone perform a test and tell me? On my side it closes the poup! so it shouldn't be call "View" but "Close" in en and "Fermer" in fr!

**Comment by ujdhesa** [comment]
> On my side it closes the poup! 
Assuming that the __View__ of the original is correct, in order to display the screenshot, closing the popup is the first thing to be expected. That you don't see the second step happening might be a sign of something broken in the software __or__ in your side. Or in both.

**Comment by ldmpub** [comment]
I agree. Can someone else can perform a test on his side? (it looks like nobody knows this software or care to this issue. So users need to perform several tests!).

It's amazing how long it could take to have a clear answer to a simple question, I re-ask: **What this button is expected to do?**

**Comment by zhuxiujuancindy** [comment]
> I agree. Can someone else can perform a test on his side? (it looks like nobody knows this software or care to this issue. So users need to perform several tests!).
> 
> It's amazing how long it could take to have a clear answer to a simple question, I re-ask: **What this button is expected to do?**

After capturing the picture, click "View" to check and have a look of that screenshot.

**Comment by ldmpub** [comment]
Thanks for your answer.
Clicking "View" button do not open the taken screenshot under Xubuntu. I have of course an image viewer installed and setup to open all images extensions (MIME association). What app is Deepin supposed to open image with?

**Comment by zhuxiujuancindy** [comment]
I use Deepin Screenshot in deepin. Click View will open the default applications for images, usually it is Deepin Image Viewer. I set Shutter as the default applications for images, then the screenshot will open in Shutter.

**Comment by ldmpub** [comment]
Thanks for your answer,

First: I have clarified french translation, see my last suggestion 'Afficher l'image' here https://www.transifex.com/linuxdeepin/deepin-screenshot/translate/#fr/$/113745852?q=text%3Aview . Can you please accept it or give my user more rights in Transifex?

Second: I should have a problem on my Xubuntu environment to open image from this popup, where can I request for help in english?

**Comment by zhuxiujuancindy** [comment]
> Thanks for your answer,
> 
> First: I have clarified french translation, see my last suggestion 'Afficher l'image' here https://www.transifex.com/linuxdeepin/deepin-screenshot/translate/#fr/$/113745852?q=text%3Aview . Can you please accept it or give my user more rights in Transifex?
> 
> Second: I should have a problem on my Xubuntu environment to open image from this popup, where can I request for help in english?

For your first question, your are French reviewer now and the translation has been accepted. But your translations will be updated in 15.9, not recently.
For your second question, I have not used Xubuntu and do not know how to solve it. Maybe you can go to https://xubuntu.org/irc/ for help.
